src/test/java/io/github/sskorol/data/DataSuppliers.java

```diff
@@ -32,4 +34,16 @@
                                 .readFrom(dataIn.source(), (Class<T>) dataIn.entity()))).toArray())
                 .orElseThrow(() -> new NoClassDefFoundError("No Data class found"));
     }
+
+    @SuppressWarnings("unchecked")
+    @DataSupplier(flatMap = true)
+    public <T> T getDataSet(final Method method) {
+        return (T) ofNullable(method.getDeclaredAnnotation(Data.class).entity().getDeclaredFields())
```

Comment:
try to decompose this line... `ofNullable` should be used for annotation checking, not for fields... if there's no annotation, you'll still get NPE...